### PR TITLE
fix: add support for overrides contexts

### PR DESCRIPTION
**Pull request by bjagg**

### CODE REVIEW

File: notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/NotifacationApplication.java [hunk 1]
```diff
@@ -23,11 +23,11 @@
 import org.springframework.context.annotation.ImportResource;
 
 @SpringBootApplication
-@ImportResource("classpath:/context/*.xml") // Legacy, XML-based beans
+@ImportResource({"classpath:/context/*.xml","classpath:/properties/contextOverrides/*.xml"}) // Legacy, XML-based beans
```
Comment (ChristianMurphy) [inline notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/NotifacationApplication.java]: The code looks good, could we rename the file from `NotifacationApplication.java` to `NotificationApplication.java`?

File: notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/NotifacationApplication.java [hunk 2]
```diff
@@ -23,11 +23,11 @@
...
 public class NotifacationApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(NotifacationApplication.class, args);
-	}
+    public static void main(String[] args) {
+        SpringApplication.run(NotifacationApplication.class, args);
```
Comment (ChristianMurphy) [inline notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/NotifacationApplication.java]:
```suggestion
        SpringApplication.run(NotificationApplication.class, args);
```

File: notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/NotifacationApplication.java (hunk 1 above)
Comment (bjagg) [reply notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/NotifacationApplication.java]: Sounds good

**Comment by jonathanmtran** [comment]
I built the portlet with this PR but I'm not seeing it pick up `overridesContext.xml`

**Comment by bjagg** [comment]
@jonathanmtran how did you test it?

**Comment by jonathanmtran** [comment]
- Deployed a snapshot build of this repo
- Moved my beans from `applicationContext.xml` to `overlays/NotificationPortlet/src/main/resources/properties/contextOverrides/overridesContext.xml` and removed `overlays/NotificationPortlet/src/main/resources/context/applicationContext.xml`

**Comment by bjagg** [comment]
I added a bean to the Notification Portlet's overrides context XML file and saw it in the log.

I'll test the uPortal-start approach

**Comment by bjagg** [comment]
Can you confirm that the file ends up at ...

```
.gradle/tomcat/webapps/NotificationPortlet/WEB-INF/classes/properties/contextOverrides/overridesContext.xml
```

I was able to see my beans deployed. I added the file at:

```
overlays/NotificationPortlet/src/main/resources/properties/contextOverrides/overridesContext.xml
```

**Comment by jonathanmtran** [comment]
Embarrassingly, the issue was due to the fact that I was deploying the wrong version (pretty sure I checked). The PR is working is as expected.

**Pull request merged**